op-e2e/actions/upgrades/interop_fork_test.go

```diff
@@ -0,0 +1,144 @@
+
+	////////////////////////////
+	// Pre-upgrade Block Production
+	////////////////////////////
+
+	// Start op-nodes
+	actors.ChainA.Sequencer.ActL2PipelineFull(t)
+	actors.ChainB.Sequencer.ActL2PipelineFull(t)
+
+	rollupConfigA := is.Out.L2s[actors.ChainA.ChainID.String()].RollupCfg
+	rollupConfigB := is.Out.L2s[actors.ChainB.ChainID.String()].RollupCfg
+
+	// Verify Interop is not active at genesis yet
+	l2Head := actors.ChainA.Sequencer.L2Unsafe()
+	require.NotZero(t, l2Head.Hash)
+	require.True(t, rollupConfigA.IsIsthmus(l2Head.Time), "Isthmus should be active at genesis in chain A")
+	require.False(t, rollupConfigA.IsInterop(l2Head.Time), "Interop should not be active at genesis in chain A")
+
+	l2Head = actors.ChainB.Sequencer.L2Unsafe()
+	require.NotZero(t, l2Head.Hash)
+	require.True(t, rollupConfigB.IsIsthmus(l2Head.Time), "Isthmus should be active at genesis in chain B")
+	require.False(t, rollupConfigB.IsInterop(l2Head.Time), "Interop should not be active at genesis in chain B")
+
+	// Submit all and mine for both chains
+	actors.ChainA.Sequencer.ActL2EmptyBlock(t)
+	actors.ChainB.Sequencer.ActL2EmptyBlock(t)
+
+	currentBlockHeaderA := actors.ChainA.SequencerEngine.L2Chain().CurrentBlock()
+	currentBlockHeaderB := actors.ChainB.SequencerEngine.L2Chain().CurrentBlock()
+	require.False(t, rollupConfigA.IsInterop(currentBlockHeaderA.Time),
+		"Interop should not be active at the first L1 inclusion block in chain A")
+	require.False(t, rollupConfigB.IsInterop(currentBlockHeaderB.Time),
+		"Interop should not be active at the first L1 inclusion block in chain B")
+
+	// Build a few L2 blocks. We only need the L1 inclusion to advance past Interop and Interop
+	// shouldn't activate with L2 time.
+	actors.ChainA.Sequencer.ActBuildL2ToInterop(t)
+	actors.ChainB.Sequencer.ActBuildL2ToInterop(t)
+
+	////////////////////////////
+	// Post-upgrade Block Production
+	////////////////////////////
+
+	activationBlockHeaderA := actors.ChainA.SequencerEngine.L2Chain().CurrentBlock()
+	activationBlockHeaderB := actors.ChainB.SequencerEngine.L2Chain().CurrentBlock()
+	require.Equal(t, activationBlockHeaderA.Number.Uint64(), activationBlockHeaderB.Number.Uint64())
+	activationBlockNumber := activationBlockHeaderA.Number
+
+	require.True(t, rollupConfigA.IsInteropActivationBlock(activationBlockHeaderA.Time),
+		"Interop should be active at the first L1 inclusion block in chain A")
+	require.True(t, rollupConfigB.IsInteropActivationBlock(activationBlockHeaderB.Time),
+		"Interop should be active at the first L1 inclusion block in chain B")
+
+	activationBlock := actors.ChainA.SequencerEngine.L2Chain().GetBlockByHash(activationBlockHeaderA.Hash())
+	activationBlockTxs := activationBlock.Transactions()
+	VerifyInteropContractsDeployedCorrectly(t, actors, activationBlockTxs, activationBlockNumber)
+
+	actors.ActBatchAndMine(t, actors.ChainA, actors.ChainB)
+
+	actors.ChainA.Sequencer.ActL2PipelineFull(t)
+	actors.ChainB.Sequencer.ActL2PipelineFull(t)
```

Comment:
I think we need an `ActL1HeadSignal` before running these or they won't derive including the latest L1. But I see in the for loop below that this is happening via the supervisor with `actors.Supervisor.SignalLatestL1(t) `. So we can just delete these lines.